src/Symfony/Bridge/Twig/Mime/TemplatedEmail.php

```diff
@@ -100,15 +100,15 @@ public function markAsRendered(): void
      */
     public function __serialize(): array
     {
-        return [$this->htmlTemplate, $this->textTemplate, $this->context, parent::__serialize()];
+        return [$this->htmlTemplate, $this->textTemplate, $this->context, $this->locale, parent::__serialize()];
     }
 
     /**
      * @internal
      */
     public function __unserialize(array $data): void
     {
-        [$this->htmlTemplate, $this->textTemplate, $this->context, $parentData] = $data;
+        [$this->htmlTemplate, $this->textTemplate, $this->context, $this->locale, $parentData] = $data;
```

Comment:
```suggestion
        [$this->htmlTemplate, $this->textTemplate, $this->context, $this->locale, $parentData] = $data;
        $this->locale = $data[5] ?? null;
`````

Reply:
Thank you, but the 5 should be a 4, right?
And is the assignment in the line before still necessary then?

```suggestion
        [$this->htmlTemplate, $this->textTemplate, $this->context, $parentData] = $data;
        $this->locale = $data[4] ?? null;
```

Reply:
correct it should be 4